aspnetcore/signalr/java-client.md

```diff
@@ -69,6 +69,15 @@ SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further detail
 
 This can safely be ignored.
 
+## Android development notes
+
+With regards to Android SDK compatibility for the SignalR client features, consider the following items when specifying your target Android SDK version:
+
+* The SignalR Java client runs on **Android API Level 16 and above**.
```

Comment:
and above --> and later

in reference to software versions

Reply:
thanks! fixed now.